File: src/main/java/toast/client/commands/cmds/Suffix.java
package toast.client.commands.cmds;

import toast.client.commands.Command;
import toast.client.modules.misc.CustomChat;
import toast.client.utils.FancyChatUtil;
import toast.client.utils.Logger;

import java.util.regex.Matcher;
import java.util.regex.Pattern;

public class Suffix extends Command {
    public Suffix() {
        super("suffix [suffix]", "Set custom chat ending", false, "suffix", "sufx");
    }

    Pattern withinQuotes = Pattern.compile("([\"'])(?:(?=(\\\\?))\\2.)*?\\1");
    Matcher matcher;
    String out;

    @Override
    public void run(String[] args) {
        if(mc.player == null) return;
        if(args.length >= 1) {
            if (!args[0].isEmpty()) {
                StringBuilder message = new StringBuilder();
                for(int i = 0; i < args.length; i++){
                    if (i > 0) message.append(" ");
                    message.append(args[i]);
                    out = message.toString();
                }
                Logger.message("Set suffix to " + out, Logger.INFO);
                CustomChat.suffix = out;
            }
        }else { Logger.message("Specify suffix!", Logger.ERR); }
    }
}

File: src/main/java/toast/client/commands/cmds/Suffix.kt
package toast.client.commands.cmds

import toast.client.ToastClient
import toast.client.commands.Command
import toast.client.modules.misc.CustomChat
import toast.client.utils.MessageUtil

/**
 * Command to change the CustomChat suffix
 */
class Suffix : Command("Suffix", """${ToastClient.cmdPrefix}suffix [suffix]""", "Set custom chat ending", false, "suffix", "sufx") {
    var out: String = ""
    override fun run(args: Array<String>) {
        if (mc.player == null) return
        if (args.isNotEmpty()) {
            val message = StringBuilder()
            for (i in args.indices) {
                if (i > 0) message.append(" ")
                message.append(args[i])
                out = message.toString()
            }
            MessageUtil.sendMessage("Set suffix to $out", MessageUtil.Color.GREEN)
            CustomChat.suffix = out
        } else {
            MessageUtil.sendMessage("Specify suffix!", MessageUtil.Color.RED)
        }
    }
}